static_src/components/users.jsx

```diff
@@ -116,75 +122,85 @@ export default class Users extends React.Component {
     return entityGuid;
   }
 
-  _onChange() {
-    this.setState(stateSetter());
+  get currentUserIsOrgManager() {
+    const { currentUser } = this.state;
+    const { currentOrgGuid } = OrgStore;
+
+    // TODO this is not 100% robust.
+    return UserStore.hasRole(currentUser, currentOrgGuid, 'org_manager');
   }
 
-  render() {
-    let removeHandler;
+  get notification() {
+    const { inviteNotices } = this.state;
 
-    if (this.state.currentType === ORG_NAME) {
-      removeHandler = this.handleRemove;
-    }
+    if (!inviteNotices.description) return null;
 
-    let content = (<UserList
-      users={ this.state.users }
-      userType= { this.state.currentType }
-      entityGuid={ this.state.entityGuid }
-      currentUserAccess={ this.state.currentUserAccess }
-      empty={ this.state.empty }
-      loading={ this.state.loading }
-      saving={ this.state.isSaving }
-      onRemove={ removeHandler }
-      onAddPermissions={ this.handleAddPermissions }
-      onRemovePermissions={ this.handleRemovePermissions }
-    />);
-
-    let notification;
-    let userInvite;
-
-    if (this.state.userListNotices.description) {
-      const notice = this.state.userListNotices;
-      notification = (
-        <Notification
-          message={ notice.description }
-          actions={ [] }
-          onDismiss={ this.onNotificationDismiss }
-          status={ notice.noticeType }
-        />
+    return (
+      <Notification
+        message={ inviteNotices.description }
+        actions={ [] }
+        onDismiss={ this.onNotificationDismiss }
+        status="finish"
+      />
+    );
+  }
+
+  get userInvite() {
```

Comment:
It will, I need to add the `entityType` interpolation we will in the `UserInvite` component. I might just move this to a separate component so there isn't a ton of logic in the method.